iocore/net/UnixNetAccept.cc

```diff
@@ -470,6 +470,7 @@ NetAccept::acceptFastEvent(int event, void *ep)
     } else {
       close_UnixNetVConnection(vc, e->ethread);
     }
+    vc = nullptr;
```

Comment:
@masaori335 https://github.com/oknet/trafficserver/blob/065c4a25f2b4c308127d08df0af65e0fc5a1cb19/iocore/net/UnixNetAccept.cc#L426-L429

goto Lerror then close the netvc that is created in the previous loop.